.github/ISSUE_TEMPLATE.md

```diff
@@ -7,14 +7,11 @@
 ### Plugin version
```

Comment:
Suggest matching the other headings by adding **(Bug -or- Feature Request Only)**